ui/src/app/admin/controllers/AdminCtrl.js

```diff
@@ -1,67 +1,67 @@
-(function(){
+(function () {
   'use strict';
   angular
     .module('app')
-    .controller('AdminCtrl', [ '$scope', 'ToastService', 'AdminService', 'UtilityService', AdminCtrl]);
+    .controller('AdminCtrl', ['$scope', 'ToastService', 'AdminService', 'UtilityService', AdminCtrl]);
 
   function AdminCtrl($scope, ToastService, AdminService, UtilityService) {
 
     $scope.signup_requests = [];
     $scope.usageChartData = [];
 
     $scope.chartOptions = {
-        chart: {
-            type: 'pieChart',
-            height: 510,
-            donut: true,
-            x: function (d) { return d.user_name; },
-            y: function (d) { return d.size; },
-            valueFormat: (d3.format('.0f')),
-            color: ['rgb(0, 150, 136)', '#E75753'],
-            showLabels: false,
-            showLegend: false,
-            title: 'Usage statistics',
-            margin: { top: -10 },
-            tooltips: true,
-            tooltip: {
-              contentGenerator: function(d) { return d.data.user_name + ' (' + UtilityService.formatBytes(d.data.size) + ')'; }
-            }
+      chart: {
+        type: 'pieChart',
+        height: 510,
+        donut: true,
+        x: function (d) { return d.user_name; },
+        y: function (d) { return d.size; },
+        valueFormat: (d3.format('.0f')),
+        color: ['rgb(0, 150, 136)', '#E75753'],
+        showLabels: false,
+        showLegend: false,
+        title: 'Usage statistics',
+        margin: { top: -10 },
+        tooltips: true,
+        tooltip: {
+          contentGenerator: function (d) { return d.data.user_name + ' (' + UtilityService.formatBytes(d.data.size) + ')'; }
         }
+      }
     };
 
-    $scope.start = function() {
+    $scope.start = function () {
       AdminService.startDownloads().then(function (response) {
         ToastService.showToast('Downloading');
-      }, function(error){
+      }, function (error) {
         ToastService.showToast('Oops! Something went wrong');
       });
     };
 
-    $scope.kill = function() {
+    $scope.kill = function () {
       AdminService.killDownloads().then(function (response) {
         ToastService.showToast('Paused all downloads');
-      }, function(error){
+      }, function (error) {
         ToastService.showToast('Oops! Something went wrong');
       });
     };
 
-    $scope.approve = function(user) {
+    $scope.approve = function (user) {
       AdminService.approve(user.user_name).then(function (response) {
```

Comment:
@malithsen are we only intended to approve a request, not to deny because in Admin service and Backend we haven't written any endpoint to deny a signup request. So should we create an endpoint or let's leave it or now?

Reply:
@vivonk I did create a denial system using the /block endpoint. It's in another PR